tools/cli/admin_db_clean_command.go

```diff
@@ -131,15 +132,19 @@ func fixExecution(
 	invariants []executions.InvariantFactory,
 	execution *store.ScanOutputEntity,
 ) (invariant.ManagerFixResult, error) {
-	execManager, err := getDeps(c).initializeExecutionStore(c, execution.Execution.(entity.Entity).GetShardID())
-	defer execManager.Close()
+	execManager, err := getDeps(c).initializeExecutionManager(c, execution.Execution.(entity.Entity).GetShardID())
+	if execManager != nil {
```

Comment:
nit: is there a chance of having error == nil and execManager == nil?
If yes - why?
If no:
Usual pattern in go is:
```
stuff, err := CreateStuff()
if err != nil {
  // handle error
}
defer stuff.Close()
or 
defer func(){
   err = stuff.Close
   if err != nil {
     // handle error
   }
}
```
I'm unsure about adding extra checks for nil make sense